docs/FAQ

```diff
@@ -1318,7 +1319,7 @@ FAQ
   you want to change name resolver function you must rebuild libcurl and tell
   it to use a different function.
 
-  - The non-IPv6 resolver that can use one out of four host name resolve calls
+  - The non-IPv6 resolver that can use one out of four host names resolves calls
```

Comment:
I see what you are saying "--resolve" option.
Changed to: "  - The non-IPv6 resolver that can use one of four different host name resolve 
  calls (depending on what your system supports):"